RepositoryCreator.py

```diff
@@ -71,6 +71,25 @@ def addPlanetsFromXML(self, planetRoots) -> None:
                     ability_name = "No Shipyard Defined"
                 newplanet.shipyardLevel = ability_name
 
+                ability_name = self.__xml.getObjectProperty(name, planetRoot, ".//Encyclopedia_Weather_Name")
+                if ability_name == "TEXT_RESOURCE_SUPPORTS_CLONING":
```

Comment:
This would be better as a dict, but if you don't know what those are, that's fine

Reply:
They are very similar to Lua tables